### PR TITLE
Fix initial staking in one-chain script

### PR DESCRIPTION
**Pull request by vitorenesduarte**
## Description

The `one-chain` script is not working after upgrading to the latest gaia version. It seems that now `gaiad gentx validator ...` is expecting only stake as argument, and fails if we also pass it some samoleans.

This PR fixes the above issue and tries to bring `dev-env` and `one-chain` together, style-wise.

I've also added two commands to the end of `one-chain` that show how to query the balance with `gaiad query bank balances ...`. This command seems to ignore the `--home` argument (used in all the other commands) and only works properly if we pass a `--node` argument pointing to the RPC interface of the Tendermint node.

______

For contributor use:

- [ ] Updated the __Unreleased__ section of [CHANGELOG.md](https://github.com/informalsystems/ibc-rs/blob/master/CHANGELOG.md) with the issue.
- [ ] If applicable: Unit tests written, added test to CI.
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Updated relevant documentation (`docs/`) and code comments.
- [ ] Re-reviewed `Files changed` in the Github PR explorer.

### CODE REVIEW

**Comment by vitorenesduarte** [comment]
With this, the instructions in https://github.com/informalsystems/ibc-rs/blob/master/relayer-cli/relayer_operation_instructions.md no longer work. Checking why.

**Comment by vitorenesduarte** [comment]
The instructions are working after 7f23e72815f66ba9cfadb19dd7d6f87262f54f70. This commit changes the `one-chain` script so that:
- the user's genesis account is the one created first (it was originally like this; I don't know why it doesn't work the other way around)
- the user has stake to perform actions

File: dev-env [hunk 1]
```diff
@@ -72,31 +72,41 @@ mkdir -p "$GAIA_DATA" && cd "$GAIA_DATA" && cd ../
...
-"$ONE_CHAIN" gaiad "$chainid1" ./data 26557 26556 6061 9091
+CHAIN_0_RPC_PORT=26657
+CHAIN_0_RPC_ADDR="localhost:$CHAIN_0_RPC_PORT"
+CHAIN_1_RPC_PORT=26557
+CHAIN_1_RPC_ADDR="localhost:$CHAIN_1_RPC_PORT"
 
-[ -f "$GAIA_DATA/$chainid0.log" ] && echo "$chainid0 initialized. Watch file $GAIA_DATA/$chainid0.log to see its execution."
-[ -f "$GAIA_DATA/$chainid1.log" ] && echo "$chainid1 initialized. Watch file $GAIA_DATA/$chainid1.log to see its execution."
+CHAIN_0_SAMOLEANS=100
+CHAIN_1_SAMOLEANS=100
```
Comment (ancazamfir) [inline dev-env:81]: Can you increase this to something like `100000000000`. 
`100` is not enough for packet testing requirements.

**Review by ancazamfir — COMMENTED** [review]
Looks good, one comment on the initial account balance.

**Review by ancazamfir — APPROVED** [review]
Great! Thanks Vitor!!

**Pull request merged**